### DISCUSSION TITLE
Humedal/zona pantanosa

### DISCUSSION THREAD

**Discussion by agustincaballero** [manudona92/MapantES#36] [General]
Cerca de la cuadrícula 304944630 hay una zona que el IGN tiene catalogada como Humedal:

![image](https://user-images.githubusercontent.com/18677366/107146422-664fce00-6948-11eb-9a2b-fa91e48eafb1.png)

Habría que ver si queremos dibujar este elemento, no sé cómo de homogéneo será en todo el territorio.
En caso de decidir ponerlo, habría que decidir cuál de los símbolos de la ISOM 2017 queremos usar, 308.0 ó 310.0 parecen los más adecuados.

**Comment by manudona92** [comment]
Esa capa Humedal es diferente a Marisma, verdad? En tal caso, para diferenciarlas, Humedal utilizaria 310 y Marisma 308

Aún asi en el ejemplo que pones, la BTN25 no renderiza esa capa (?), al menos no aparece ningún símbolo azul en el mapa.

**Reply by agustincaballero** [reply]
Están ambas en el fichero 0319S de la BTN, y los distingue por un atributo:

![image](https://user-images.githubusercontent.com/18677366/107151781-b3da3400-6964-11eb-9608-8f7eb10439b6.png)

El IGN parece que no renderiza el humedal, la marisma parece que sí (zona de Torrevieja, por ejemplo)

**Reply by manudona92** [reply]
En ese caso, según la descripción, para ZONA ENCHARCABLE utilizaria el 310. Marisma el 308. 

Ambos símbolos combinan bien con el resto de colores. El único inconveniente es que puede el arroyo o rio pierde visibilidad si la mancha del humedal queda por encima de este. Pero en términos generales, me atrevería a incluirlo y ver.
